.circleci/config.yml

```diff
@@ -3310,6 +3310,7 @@ workflows:
             branches:
               only:
                 - main
+                - /release-(\d)+(\.(\d)+)+-ee/
```

Comment:
is the `ee` part optional here?

Reply:
Also, would this pipeline trigger upon creation or the release branch, or would it wait until a scheduled time? Seems like we'd want this workflow to happen ASAP.

Reply:
You tell me? :-)

Reply:
Updated.

Reply:
Upon testing, the `ee` part of regex is mandatory, so probably something like `release-(\d)+(\.(\d)+)+(-ee)?$` instead.